CHANGELOG.rst

```diff
@@ -10,6 +10,8 @@ This document describes changes between each past release.
 
 - Remove Python 3.5 support and upgrade to Python 3.6. (#1886)
 - Remove `record` from UnicityError class (#1919). This enabled us to fix #1545.
+- Storage backend API has changed, notions of collection and records were replaced
+  by the generic terms *resource* and *object*.
```

Comment:
It would (still...) nice to add a note here mentioning that this could break you if you implemented a resource subclass.